src/geovista/core.py

```diff
@@ -375,7 +376,9 @@ def combine(
         common_point_data = set(first.point_data.keys())
         common_cell_data = set(first.cell_data.keys())
         common_field_data = set(first.field_data.keys())
-        active_scalars_info = {first.active_scalars_info._namedtuple}  # noqa: SLF001
+        active_scalars_info = {
+            pv.core.dataset.ActiveArrayInfoTuple(*first.active_scalars_info)
```

Comment:
If https://github.com/pyvista/pyvista/pull/6835 is accepted and the next GeoVista release pins `pyvista>=0.45`, then `active_scalars_info` _is_ a named tuple and no unpacking is needed. Maybe this change could be made in a future PR?

```suggestion
           first.active_scalars_info
```